src/Core/src/Platform/Windows/WindowRootView.cs

```diff
@@ -223,9 +229,38 @@ void LoadAppTitleBarControls()
 				AppFontIcon.ImageFailed += OnImageFailed;
 			}
 
+			ApplyTitlebarColorPrevalence();
 			UpdateAppTitleBarMargins();
 		}
 
+		private void ViewSettingsColorValuesChanged(ViewManagement.UISettings sender, object args)
+		{
+			ApplyTitlebarColorPrevalence();
+		}
+
+		void ApplyTitlebarColorPrevalence()
+		{
+			try
+			{
+				// Figure out if the "show accent color on title bars" setting is enabled
+				using var dwmSubKey = Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\Windows\DWM\");
```

Comment:
I'm not sure if we have an established way to do registry stuff, but from what I know this is the only way to check for the accent color setting 😕